File: MonopolyDealGame/MonopolyDealGame/DiscardPile.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MonopolyDealGame
{
    class DiscardPile:Deck
    {
        public DiscardPile(Table gameTable, List<Card> cards)
        { 
        }

        public DiscardPile(Table gameTable)
        {
        }

        public void draw()
        {

        }
        public override string ToString()
        {
            return base.ToString();
        }
        public override String getDeckName() 
        {
            return "Discard Pile";
        }
    }
}

File: Monopoly Deal/src/cards/DiscardPile.java
package cards;

import java.util.List;

import board.Table;

public class DiscardPile extends Deck {
	public DiscardPile(Table gameTable, List<Card> cards) {
	}

	public DiscardPile(Table gameTable) {
	}

	public void draw() {

	}

	public String toString() {
		return super.toString();
	}

	public String getDeckName() {
		return "Discard Pile";
	}

}